index.html

```diff
@@ -180,30 +180,33 @@
       </div>
       <div class="row">
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Install Bazel</h3>
-          <p>See our <a href="docs/install.html">Installation Guide</a> to learn
-             how to download and install Bazel.
+          <h3>Java</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/java.html">Build Java tutorial</a>.
           </p>
         </div>
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Run Bazel</h3>
-          <p>See our
-             <a href="docs/getting-started.html">Getting Started Guide</a> to
-             learn how to write a BUILD file and run Bazel.
+          <h3>C++</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/cpp.html">Build C++ tutorial</a>.
           </p>
         </div>
       </div>
       <div class="row">
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Bazel Commands</h3>
-          <p>See our <a href="docs/bazel-user-manual.html">User Manual</a> to
-             learn about the Bazel command-line tool.
+          <h3>Android</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/app.html">mobile app tutorial</a>.
           </p>
         </div>
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Learn More</h3>
-          <p>Take a look at our <a href="docs/install.html">documentation</a>
-             and read our <a href="faq.html">FAQ</a>.
+          <h3>iOS</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/app.html">mobile app tutorial</a>.
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-4 col-sm-offset-4">
+          <h3>Other languages</h3>
+          <!-- Link to GitHub until we have a better {{ site.docs_site_url }}/be/overview.html-->
```

Comment:
> Ok I am going to delete the 'other languages' to make progress, even if I personally think there is a very high value to show users the tremendous list of language rules.

Thanks, Steren. Definitely agree that there is very high value in showcasing Bazel's vast language support. Once the new website is launched, let's prioritize the work that will help build and improve the experience of Bazel's ecosystem, such as bazelbuild/bazel#3255 and building a one-stop-shop for browsing packages as we have discussed, which would solve the problem for both third-party libraries as well as rules (bazelbuild/bazel#1046).